pkgs/data/misc/tzdata/default.nix

```
@@ -42,10 +42,11 @@ stdenv.mkDerivation rec {
 
   installFlags = [ "ZIC=./zic-native" ];
 
+  # NOTE: in cross-compilation scenarios, this is built for the build platform.
   preInstall = ''
      mv zic.o zic.o.orig
      mv zic zic.orig
-     make $makeFlags cc=${stdenv.cc.targetPrefix}cc AR=${stdenv.cc.targetPrefix}ar zic
+     make $makeFlags cc=cc AR=ar zic
```

```suggestion
     # NOTE: in cross-compilation scenarios, this is built for the build platform.
     make $makeFlags cc=cc AR=ar zic
```
just to make it super clear.